### PR TITLE
Added keyval_to_unicode

### PR DESCRIPTION
**Pull request by trolleyman**
Maybe this will work this time! :P

### CODE REVIEW

File: src/keys.rs [hunk 1]
```diff
@@ -7,9 +7,17 @@
...
 
 pub fn keyval_name(keyval: u32) -> Option<String> {
     unsafe {
         from_glib_none(ffi::gdk_keyval_name(keyval as c_uint))
     }
 }
+
+pub fn keyval_to_unicode(keyval: u32) -> Option<char> {
+	unsafe {
+		let c = char::from_u32(ffi::gdk_keyval_to_unicode(keyval));
```
Comment (GuillaumeGomez) [inline src/keys.rs]: We use spaces tabulation (' ') not tab tabulations ('\t'). On top of that, your indentation isn't good. Not good ! =D

File: src/keys.rs [hunk 2]
```diff
@@ -7,9 +7,17 @@
...
+		if c == Some('\0') {None} else {c}
```
Comment (GuillaumeGomez) [inline src/keys.rs]: Put spaces insides brackets :

```Rust
if c == Some('\0') { None } else { c }
```

**Comment by GuillaumeGomez** [comment]
Fix the issues I mentioned and please squash like I explained [here](https://github.com/rust-gnome/gdk/pull/47#issuecomment-104252508).

**Comment by trolleyman** [comment]
Arg, thought I was on spaces :P

**Comment by GuillaumeGomez** [comment]
I understand, I had the same issues when I started. ^^

**Pull request closed without merging**